src/mito2/src/manifest/action.rs

```diff
@@ -22,13 +22,14 @@ use snafu::{OptionExt, ResultExt};
 use store_api::manifest::ManifestVersion;
 use store_api::metadata::RegionMetadataRef;
 use store_api::storage::{RegionId, SequenceNumber};
+use strum::Display;
```

Comment:
```suggestion
```

Reply:
I'd like to keep the `Display` for enum, it does no harm but maybe useful in the future like in the logging.

Reply:
Though we can use `Debug`.......